prep-javascript-conditionals/main.js

```diff
@@ -0,0 +1,41 @@
+/* Your JS goes in this file */
+function isAdult(age) {
+  return age < 18 ? false : true;
+}
```

Comment:
Ternary not even necessary in this situation! Anytime you do a comparison operator, it will always evaluate to `true` or `false.`

So you could just `return age < 18` and it will automatically return `true` or `false`